metaseq/checkpoint_utils.py

```diff
@@ -95,6 +95,7 @@ def save_checkpoint(
             extra_state,
             training_finished=training_finished,
             async_callback_fn=async_callback_fn if save_to_NFS else None,
+            files_to_symlink_to=checkpoints[1:] if len(checkpoints) > 1 else None,
```

Comment:
can use `functools.partial` instead

Reply:
async_callback_fn is already a partial, so I could partial the partial to fill in the `files_to_symlink_to` arg first but that code will be very sad to follow...

Reply:
Ya, that's what i meant by partial the partial, but seems like we don't gain much from readability either way...

Reply:
I'm more worried about interaction with async copying logic right now, since I think that's currently skipped in the Azure setup if you're only using local saves. Will test that out and update here!